sdk/create-dapp/templates/react-client-dapp/src/App.tsx

```diff
@@ -0,0 +1,39 @@
+import { ConnectButton } from "@mysten/dapp-kit";
+import { Box, Container, Flex, Heading } from "@radix-ui/themes";
+import { WalletStatus } from "./WalletStatus";
+
+function App() {
+  return (
+    <>
+      <Flex
+        position="sticky"
+        px="4"
+        py="2"
+        justify="between"
+        style={{
+          borderBottom: "1px solid var(--gray-a2)",
+        }}
+      >
+        <Box>
+          <Heading>dApp Starter Template</Heading>
+        </Box>
+
+        <Box>
+          <ConnectButton />
```

Comment:
it would be really cool to theme the button based on the selected Radix Theme (e.g. using the set of accent color variables defined by their color system), but I know that would require some extra effort and might complicate the example 😅

Reply:
I think this would be great to do, but let's push it to another PR,  I'm in a bit of a time crunch this week, and want to get this out in time to reference it in some docs I need to write before the offsite